test/libc/fmt/printf_uppercase_float_static_yoink_test.c

```diff
@@ -0,0 +1,45 @@
+/*-*- mode:c;indent-tabs-mode:nil;c-basic-offset:2;tab-width:8;coding:utf-8 -*-│
+│vi: set net ft=c ts=2 sts=2 sw=2 fenc=utf-8                                :vi│
+╞══════════════════════════════════════════════════════════════════════════════╡
+│ Copyright 2021 Justine Alexandra Roberts Tunney                              │
```

Comment:
As this is your own test file, you can have the copyright for the file: Maybe  `Copyright 2023 Gabriel Ravier`

Reply:
I guess I'll take it, then, yeah.